divi/src/test/test_divi.cpp

```diff
@@ -34,7 +34,6 @@ extern CCoinsViewCache* pcoinsTip;
 extern bool fCheckBlockIndex;
 extern BlockMap mapBlockIndex;
 extern int nScriptCheckThreads;
-extern CChain chainActive;
```

Comment:
The other globals above could potentially be removed here, even if it's a bit tricky to smack the compiler around till it yields

Reply:
Ah, it's because of the shallow db initializations isnt it?

Reply:
This file, `init` and the `ChainstateManager` itself are the only places left where the globals are directly accessed.  The former two are where they are set up.  I think it makes sense to first remove all the usages, and work on the set up in a future step (e.g. moving the setup both in `init` and the unit tests inside the `ChainstateManager` class, or something like this).

Reply:
The change made here was trivial (as you can see), that's why I did it, but I did not want to follow up on the other globals for the reason stated above.  If you want me to completely revert this change for instance, I can do that of course.